firmware/application/ui_navigation.cpp

```diff
@@ -689,9 +689,21 @@ ReceiversMenuView::ReceiversMenuView(NavigationView& nav) {
 
     add_apps(nav, *this, RX);
 
-    for (auto const& gridItem : ExternalItemsMenuLoader::load_external_items(app_location_t::RX, nav)) {
-        add_item(gridItem);
-    };
+    auto externalItems = ExternalItemsMenuLoader::load_external_items(app_location_t::UTILITIES, nav);
+    if (externalItems.empty()) {
+        add_item({"More",
```

Comment:
I don't really like "More" but have no better proposal... looking for some short form that could express that "there could be more if you copy it to the sd" instead of creating the expectation that you get a submenu with more apps on click :)

Reply:
Changed to “Notice”